atomix/cluster/src/main/java/io/atomix/raft/impl/zeebe/ZeebeRaftStateMachine.java

```diff
@@ -41,7 +41,7 @@ public ZeebeRaftStateMachine(final RaftContext raft) {
     this.lastEnqueued = reader.getFirstIndex() - 1;
     this.logger =
         ContextualLoggerFactory.getLogger(
```

Comment:
Nit: as you mentioned, we could also just add the partition into the MDC and use a normal logger here. Up to you.

Reply:
Will probably do during some of the refactorings, let see